gatewayconfig/protos/add_gateway_pb2.py

```diff
@@ -1,89 +1,91 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't understand why these files change at all. Most changes appear to be cosmetic, but I haven't done a thorough audit.

Reply:
I think this is either Python 2.7 stuff, or editor stuff.